boilerpy3/extractors.py

```diff
@@ -28,7 +28,7 @@ class Extractor:
     
     SCRIPT_REGEX = re.compile(r'<(?:script|SCRIPT)[^>]*>.*?</(?:script|SCRIPT)>', re.DOTALL)
     
-    def __init__(self, filtr: BoilerpipeFilter, raise_on_failure: bool = True) -> None:
+    def __init__(self, filtr: BoilerpipeFilter, raise_on_failure: bool = True, headers: dict = {}) -> None:
```

Comment:
As mentioned in my comment, let's change this to `request_kwargs`.

```suggestion
    def __init__(self, filtr: BoilerpipeFilter, raise_on_failure: bool = True, **request_kwargs: dict) -> None:
```

Reply:
Make sure to also update the docstrings to add the new parameter.

Reply:
Finally, please add a test for usage of this parameter.